recipe/meta.yaml

```diff
@@ -51,6 +57,9 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - openssh  # [mpi == 'openmpi']
+    - tempest-remap  # [tempest == 'tempest']
+    - libnetcdf * {{ mpi_prefix }}_*  # [tempest == 'tempest']
+    - eigen  # [tempest == 'tempest']
```

Comment:
I believe Eigen3 consists only of header files (no libraries) so it should only be needed at build.  @vijaysm and @gonuke, is that correct?

Reply:
That is correct